substrate/client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -53,6 +53,10 @@ use std::{
 /// `Initialized` event.
 const MAX_FINALIZED_BLOCKS: usize = 16;
 
+/// Maximum amount of events buffered by submit_events
+/// before dropping the stream.
+const MAX_BUFFERED_EVENTS: usize = 512;
```

Comment:
nit: I think we can set this to `MAX_PINNED_BLOCKS` or use that directly

Reply:
done